src/Mod/AddonManager/manage_python_dependencies.py

```diff
@@ -389,8 +389,7 @@ def migrate_old_am_installations(cls) -> bool:
             # Nothing to migrate
             return False
 
-        if not os.path.exists(new_directory):
-            os.makedirs(new_directory)
+        os.makedirs(new_directory, mode=0o777, exist_ok=False)
```

Comment:
Why is `exist_ok=False`? Doesn't that have the opposite of the intended effect?

Reply:
I did test it on the same machine that gave the error in the first place and AddonManager started error free so not sure how that was the case but I've corrected it.